static/styles/night_mode.scss

```diff
@@ -59,6 +59,10 @@ body.night-mode {
         color: inherit !important;
     }
 
+    .message_reaction:hover ,.reaction_button:hover{
```

Comment:
Your error coming from this line. The errors flagged are stated in CircleCI report [here](https://circleci.com/gh/zulip/zulip/16261?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
- leave a space after the comma and no space before.

- A space before the `{`

- Proper indentation of the following line. 

Read [this](https://zulip.readthedocs.io/en/latest/testing/linters.html?highlight=lint) which gives information about linting Zulip uses. You can run those scripts locally before submitting a PR.

Reply:
thank you for the explanation, will take care for the proper indentation from the next time 👍 .